app-frontend/src/app/core/services/layer.service.js

```diff
@@ -207,12 +207,19 @@ export default (app) => {
                 redGamma: object.redGamma,
                 greenGamma: object.greenGamma,
                 blueGamma: object.blueGamma,
+                redMax: object.redMax,
```

Comment:
i think it's pretty possible that this whole function is unnecessary now but that's not tested